blocks/extensions/predicates.py

```diff
@@ -12,3 +12,7 @@ def __init__(self, record_name):
 
     def __call__(self, log):
         return bool(log.current_row.get(self.record_name, False))
+
+    def __eq__(self, other):
```

Comment:
It's used in a test to make sure that the appropriate `Checkpoint.add_condition` call is made.